web/src/components/TransactionHeader/TransactionHeader.tsx

```diff
@@ -0,0 +1,30 @@
+import TestCardActions from 'components/TestCard/TestCardActions';
+import {noop} from 'lodash';
+import {useTransaction} from '../../providers/TransactionRunDetail/Transaction.provider';
+import * as S from './TransactionHeader.styled';
+
+interface IProps {
+  onBack(): void;
+}
+
+const TransactionHeader = ({onBack}: IProps) => {
+  const {transaction} = useTransaction();
+  return (
+    <S.Container>
+      <S.Section>
+        <S.BackIcon data-cy="transaction-header-back-button" onClick={onBack} />
+        <div>
+          <S.Title data-cy="transaction-details-name">
+            {transaction?.name} (v{transaction?.version})
```

Comment:
@schoren @mathnogueira are we considering versioning transactions the same way we are doing for tests?